src/components/modal/__tests__/CdrModal.spec.js

```diff
@@ -26,6 +26,10 @@ describe('CdrModal.vue', () => {
       expect(wrapper.element).toMatchSnapshot();
```

Comment:
@benjag I noticed these tests were commented out and your name was on the blame. I re-enabled them in this PR, fyi